data/avro-schema/src/main/resources/avro/net/corda/data/certificates/CertificateType.avsc

```diff
@@ -0,0 +1,11 @@
+{
+  "type": "enum",
+  "name": "CertificateType",
+  "namespace": "net.corda.data.certificates",
+  "doc": "A cluster level certificate type.",
+  "symbols": [
+    "P2P",
```

Comment:
what do we want this type to reflect exactly? Is it how the certificate is used or from which part of the system it's used?

Asking because if it's the former, should we have `P2P_SESSION` and `P2P_TLS` here? or something along those lines